test/axel_f/date_test.cljc

```diff
@@ -55,3 +55,15 @@
     (t/is (= 1988
              (sut/YEAR (sut/DATE 1988 8 21))
              ((af/compile "YEAR(DATE(1988, 8, 21))"))))))
+
+(t/deftest comparable
+  (t/testing "local date"
+    (t/is ((af/compile "DATE(1988, 8, 21) > DATE(1988, 7, 21)")))
+    (t/is ((af/compile "DATE(1988, 8, 21) = DATE(1988, 8, 21)")))
+    (t/is ((af/compile "DATE(1988, 8, 21) < DATE(1988, 9, 21)")))
+    (t/is ((af/compile "TODAY() > DATE(1988, 8, 21)")))
+    (t/is ((af/compile "TODAY() > 0"))))
```

Comment:
and let's add some more tests, like 
```
DATE(2020, 02, 03, 11, 03, 29) < 1580727810
DATE(2020, 02, 03, 11, 03, 30) = 1580727810
DATE(2020, 02, 03, 11, 03, 31) = 1580727810
```

Reply:
this isn't a valid examples. atm we don't have function to instantiate datetime, only `NOW()` can give you datetime